src/quic/streams.cc

```diff
@@ -17,9 +17,11 @@
 
 namespace node {
 
+using quic::BindingData;
```

Comment:
This `using` should not be necessary

Reply:
I remove it at my local copy and see what happens in the build process.